### DISCUSSION TITLE
Weekly Sync - 29 August 2023

### DISCUSSION THREAD

**Discussion by sabineschaller** [interledger/rafiki#1770] [Weekly Sync]
**Please add agenda items below and give your weekly update in a comment. Thank you!**

## Agenda
* @sabineschaller: new call format
* @sabineschaller: recap - roadmap (@mkurapov can explain auto-peering feature)
* @sabineschaller: current developments in Open Payments
* @BlairCurrey : add code coverage?
    * decided to make an issue, and timebox potential solution, probably just to have a local command that compares coverage
    
## Overview
* Release date & status for Rafiki versions would be helpful
* More detailed integration guide would be useful (we have this as an action item to discuss during the rafiki work week)

**Comment by sabineschaller** [comment]
Update:
* main focus on https://github.com/interledger/rafiki/pull/1782
* finished OP quoting discussion and updated issues accordingly
* some PR reviews
* Mojaloop design calls
* discussing what to change in Rafiki for testnet auto-peering with @mkurapov and @AlexLakatos

**Comment by BlairCurrey** [comment]
- Working on getting the quote fee calculation into the rafiki backend https://github.com/interledger/rafiki/issues/1639
  - have the calculation logic moved and validated that it matches the current behavior in main
  - I need to handle the cross-currency case which Max raised in a slack discussion
  - I also need to update `sendAmount` to `debitAmount` field in a few more places. think i need this to get merged before renaming the open payments response obejct: https://github.com/interledger/open-payments/pull/275

**Comment by mkurapov** [comment]
- Short week last week
- Scoped out auto peering story: https://github.com/interledger/rafiki/issues/1761
- Continued work on https://github.com/interledger/rafiki/issues/1701
- Will re-prioritize (in time for the Rafiki work week) and begin the auto-peering story instead + (as well as a few other alpha v1.0.3 items), then back to decoupling ILP

**Comment by raducristianpopa** [comment]
- Rafiki
  - PR that replaces the old logo with the new one: #1801
  - PR that introduces multi-stage docker builds to reduce image size: : #1795
- Testnet
  - Finished the initial commerce frontend scaffold
  - Starting to couple the frontend to the backend
    - Products page
    - Single product page
  - Shopping cart persistence - keep the shopping cart populated even if the user is closing the browser
  - Currently working on: shopping cart page (pre-checkout)
- Will work on:
    - Finishing the shopping cart page
    - Deep diving into the checkout flow (frontend + backend)

**Comment by njlie** [comment]
* Opened PR for Interaction refactoring #1797
  * Pretty big PR so would consider breaking it up
* Start working on #1769

**Comment by AlexLakatos** [comment]
- A lot of audio quality / internet connection issues this week
- moving along with Rafiki Work Week planning

**Comment by Tymmmy** [comment]
Worked on Testnet:
- E-commerce designs\
- Display exchange rate for sending money
- Exchange currency button

**Comment by domin191013** [comment]
Worked on Testnet:

• added socket room logic to send notification to relevant users
• extract user id from iron-session token
